docs/src/app/components/pages/components/Menu/Page.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import CodeExample from '../../../CodeExample';
+import PropTypeDescription from '../../../PropTypeDescription';
+import MarkdownElement from '../../../MarkdownElement';
+
+import menuReadmeText from './README';
+import menuCode from '!raw!material-ui/lib/menus/menu';
+import menuItemCode from '!raw!material-ui/lib/menus/menu-item';
+import MenuExampleSimple from './ExampleSimple';
+import menuExampleSimpleCode from '!raw!./ExampleSimple';
+import MenuExampleDisable from './ExampleDisable';
+import menuExampleDisableCode from '!raw!./ExampleDisable';
+import MenuExampleIcons from './ExampleIcons';
+import menuExampleIconsCode from '!raw!./ExampleIcons';
+import MenuExampleSecondary from './ExampleSecondary';
+import menuExampleSecondaryCode from '!raw!./ExampleSecondary';
+import MenuExampleNested from './ExampleNested';
+import menuExampleNestedCode from '!raw!./ExampleNested';
+
+const MenuPage = () => (
+  <div>
+    <MarkdownElement text={menuReadmeText} />
+    <CodeExample code={menuExampleSimpleCode}>
+      <MenuExampleSimple />
+    </CodeExample>
+    <CodeExample code={menuExampleDisableCode}>
+      <MenuExampleDisable />
+    </CodeExample>
+    <CodeExample code={menuExampleIconsCode}>
+      <MenuExampleIcons />
+    </CodeExample>
+    <CodeExample code={menuExampleSecondaryCode}>
+      <MenuExampleSecondary />
+    </CodeExample>
+    <CodeExample code={menuExampleNestedCode}>
+      <MenuExampleNested />
+    </CodeExample>
+    <PropTypeDescription code={menuCode}/>
```

Comment:
@oliviertassinari Yes, that was something I wanted to raise an issue about. I think the CodeExample and PropDescritption should accept title props because its hard to tell what they are showing without it...